Code/Editor/Plugins/EditorAssetImporter/ImporterRootDisplay.cpp

```diff
@@ -65,8 +65,8 @@ void ImporterRootDisplay::SetSceneHeaderText(const QString& headerText)
 {
     QFileInfo fileInfo(headerText);
     ui->m_filePathText->setText(tr("<b>%1</b>").arg(fileInfo.fileName()));
-    QString fullPath = tr("<b>%1%2</b>").arg(QDir::toNativeSeparators(fileInfo.path())).arg(QDir::separator());
-    ui->m_fullPathText->setText(fullPath);
+    QString fullPath = tr("%1%2").arg(QDir::toNativeSeparators(fileInfo.path())).arg(QDir::separator());
```

Comment:
why use tr its for translation.

Reply:
We've been using TR all over to eventually support potential translation. Not a lot you'd do with a path, but the last guidance on Qt I've seen was to use TR as much as possible. I don't mind changing to QString if we're not using tr for translation anymore.

Reply:
If it was just used to display in the label, I'd be fine with the use of tr... but since it's used to open an Url later:
```
QDesktopServices::openUrl(QUrl::fromLocalFile(fullPath));
```
I would recommend not using tr (since it could potentially be used to redirect the user to an arbitrary URL by stomping a path via the translation file).